### PR TITLE
two alarms added via de yaml

### PR DESCRIPTION
**Pull request by nielsmijnster**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -21,22 +21,22 @@ provider:
             - "dynamodb:Scan"
           Resource: '*'
 
+custom:
+  alerts:
+    stages: # Optionally - select which stages to deploy alarms to
+      - dev
```
Comment (flaviostutz) [inline serverless.yml:27]: Why did you specify this? You don't want this alarm to be deployed in other stages?

**Review by flaviostutz — APPROVED** [review]
Really nice! Congrats for the hard work!

File: serverless.yml (hunk 1 above)
Comment (nielsmijnster) [reply serverless.yml:27]: i used this a lot in other attempts, but as long as we keep custom it can be removed i guess.,

Comment (nielsmijnster) [reply serverless.yml:27]: > Really nice! Congrats for the hard work!

i did, thank you very much!!

**Pull request merged**